libmscore/tempotext.cpp

```diff
@@ -236,7 +236,7 @@ QVariant TempoText::propertyDefault(P_ID id) const
       {
       switch(id) {
             case P_ID::TEMPO:             return 2.0;
-            case P_ID::TEMPO_FOLLOW_TEXT: return false;
+            case P_ID::TEMPO_FOLLOW_TEXT: return true;
```

Comment:
Sorry for the delay to come back to this, but I don't get why we need to change the default for this property. Also, changing it here but not in the constructor doesn't look elegant. Is it really needed to change the default? Why?